src/DonorProfiles/Tabs/TabsRegister.php

```diff
@@ -81,7 +81,7 @@ public function addTab( $tabClass ) {
 
 		$tabId = $tabClass::id();
 
-		if ( isset( $this->tabs[ $tabId ] ) ) {
+		if ( $this->hasTab( $tabId ) ) {
 			throw new \InvalidArgumentException( 'A tab can only be added once. Make sure there are not id conflicts.' );
```

Comment:
Likewise, we should throw a different exception type here. The above use of `InvalidArgumentException` in `addTab()` makes sense, but on line 84-85 the argument is correct and valid (type) but there is a conflict.

Also, throwing a custom exception is easy to document for what to catch.

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5577/commits/fdaa13fd9a0dd47ee65e037b271fa2e7e60caa8c